README.md

```diff
@@ -78,7 +78,7 @@ ninja && sudo ninja install
 ## 🤔 Questions
 
 If you want to ask any questions about the project, join us on our public Matrix channel at 
-[#akiraux:matrix.org](https://matrix.to/#/!kpfeTRbpocQrOFCFnJ:matrix.org) or on IRC at #akiraux on Freenode.
+[#akiraux:matrix.org](https://matrix.to/#/#akiraux:matrix.org) or on IRC at #akiraux on LiberaChat.
```

Comment:
It may be nice to make an announcement on social medias about this change @Alecaddd

Reply:
@LucileDT What change

Reply:
We haven't made any change we fixed the link after we switched to another channel

Reply:
Yes and I never saw any post explaining AkiraUX moved to LiberaChat, this correction just made me think it would be a good idea :slightly_smiling_face: